tools/offline_log_viewer/storage.py

```diff
@@ -285,7 +285,13 @@ def __search(self):
             if 'cloud_storage_cache' in ntpd:
                 continue
             head, part_ntp_id = os.path.split(ntpd)
-            [part, ntp_id] = part_ntp_id.split("_")
+            try:
+                [part, ntp_id] = part_ntp_id.split("_")
+            except ValueError:
+                # Not a partition dir: this may happen if we have a namespace dir
+                # with no partitions.
+                continue
```

Comment:
seems like a more specific test is necessary to handle cases where a topic has an underscore?

```
print(head, part_ntp_id, part, ntp_id)

data/namespace topic1_asdf topic1 asdf
```

like here I end up with topic1/asdf as the partition and ntp_id.

Reply:
This commit went away in favor of 906a9e70 fix for empty dirs upstream